client/blocks/site/style.scss

```diff
@@ -71,8 +71,15 @@
 	position: relative;
 	width: 100%;
 
-	&:focus {
+	.accessible-focus &:focus {
 		outline: none;
+		box-shadow: inset 0 0 0 2px var( --color-primary-light );
+
+		&::after {
+			top: 2px;
+			right: 2px;
+			bottom: 2px;
+		}
```

Comment:
What is the purpose of this `::after` element? The "Elements" devtool doesn't show it at all. These pseudo-element only become active when they have a `content` attribute, so this CSS in its current form seems to be a noop.

Reply:
Thanks! Removed in: https://github.com/Automattic/wp-calypso/commit/dd1f099f8760ed332fc504be86b7c927f3ea41b9

Reply:
Thanks! Can you also add a comment explaining that this CSS is overriding the `long-content-fade`? It's hard to figure out otherwise.

Also, `display: none` should do the same job as `z-index: -1` and is more semantic.

Reply:
Can do! Updated in: https://github.com/Automattic/wp-calypso/commit/c244bf386d046c901ca27fb8b6a874466dbfedf4